src/components/Fields/TextField.js

```diff
@@ -261,7 +287,8 @@ class TextField extends Component {
     }
 
     render() {
-        const hasValue = this.state.value === 0 ? true : Boolean(this.state.value);
+        const hasValue = this.state.isAutofilled ||
+            (this.state.value === 0 ? true : Boolean(this.state.value));
```

Comment:
`x ? true : y` should reduce to `x || y` unless I'm missing something

Reply:
Ah, fair point. But I've already merged :/